bin/sdk/gutenberg.js

```diff
@@ -45,6 +47,9 @@ exports.compile = args => {
 	};
 
 	const name = path.basename( path.dirname( options.editorScript ).replace( /\/$/, '' ) );
+	const styleFile = options.outputEditorFile
+		? options.outputEditorFile.replace( '.js', '.css' )
```

Comment:
can we not perform this chance from inside of Calypso's main config file? seems like we probably wouldn't want `.js.css` anyway, right?

Reply:
Great idea! This only means we'll pass the file name without extension as a `--output-editor-file` and `--output-view-file` - for example `block` and not `block.js`.

Updated in https://github.com/Automattic/wp-calypso/pull/26527/commits/8fffcc0e559b592344f0995116b42a50386d395a, feel free to have a another look @dmsnell